scripts/upload-release.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+set -x
+SELF="$0"
+TAG="$1"
+TOKEN="$2"
+
+NAME="ksh"
```

Comment:
This var isn't used so either there is someplace it was meant to be used or it should be removed.